### PR TITLE
unshield: 1.3 -> 1.4.2

### PR DESCRIPTION
**Pull request by ndowens**
###### Motivation for this change
Update

###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Review by matthiasbeyer — APPROVED** [review]

File: pkgs/tools/archivers/unshield/default.nix [hunk 1]
```diff
@@ -1,30 +1,23 @@
...
-    })
-  ];
 
   nativeBuildInputs = [ cmake ];
   buildInputs = [ zlib openssl ];
 
   meta = with stdenv.lib; {
     description = "Tool and library to extract CAB files from InstallShield installers";
-    homepage = https://github.com/twogood/unshield;
+    homepage = "https://github.com/twogood/unshield";
```
Comment (joachifm) [inline pkgs/tools/archivers/unshield/default.nix]: Why this change?

Comment (ndowens) [inline pkgs/tools/archivers/unshield/default.nix]: Thought it would need to be wrapped in quotes

Comment (ndowens) [inline pkgs/tools/archivers/unshield/default.nix]: Fixed @joachifm

**Comment by grahamc** [comment]
Applied in 918a4e649d, thank you!
